docs/functionality/validation.md

```diff
@@ -0,0 +1,93 @@
+# Validation Service Documentation
+
+## Overview
+The **Validation Service** is a component responsible for validating FHIR resources associated with a patient 
+after the Measure Evaluation process is complete. It ensures that resources conform to both the core FHIR 
+specification and specific StructureDefinitions (profiles) as asserted in the `meta.profile` property of 
+each resource.
+
+## Process Flow
+1. **Measure Evaluation Completion**
+    - The Measure Evaluation Service evaluates a patient.
+    - Once evaluation is complete, it produces a Kafka message indicating that the patient is ready for validation.
+
+2. **Validation Service Consumption**
+    - The Validation Service consumes the Kafka message.
+    - It retrieves the **MeasureReport** for the specified patient from the Report Service.
+    - It extracts all contained FHIR resources and constructs a **Bundle** for validation.
```

Comment:
Doesn't the Report service come between these two?  Measure Eval produces `ResourceEvaluated`, but Validation consumes `ReadyForValidation`.